.werft/jobs/build/installer/installer.ts

```diff
@@ -243,6 +243,7 @@ EOF`);
     private configureUsage(slice: string) {
         exec(`yq w -i ${this.options.installerConfigPath} experimental.webapp.usage.enabled true`, { slice: slice })
         exec(`yq w -i ${this.options.installerConfigPath} experimental.webapp.usage.schedule 1m`, { slice: slice })
+        exec(`yq w -i ${this.options.installerConfigPath} experimental.webapp.usage.billInstancesAfter "2022-08-11T08:05:32.499Z"`, { slice: slice })
```

Comment:
How was this value determined? Is this value still appropriate given current contexts?

Reply:
[This](https://gitpod.slack.com/archives/C027AUU7BC3/p1660832504534679) is the context for the value.